### ISSUE TITLE
tss peers do not throw out the err when applying incorrect wiredMsg in keygen

### ISSUE THREAD

**Issue by froyobin** [binance-chain/tss-lib#98]
We have 3 nodes, A,B,C that involved in the keygen.

after successfully run the first keygen , node A saved the `wiredBytes` from the previous keygen for the round  `binance.tss-lib.ecdsa.keygen.KGRound2Message2`.

They run the keygen with same parties again.

in the round  ` "binance.tss-lib.ecdsa.keygen.KGRound2Message2"` node A broadcasts the `wiredBytes` that from the previous keygen request to the peers, and the peers accept this incorrect  `wiredBytes` as `partyInfo.Party.UpdateFromBytes` return nil error. Since the peers applied the incorrect share, their `outChannel` cannot produce the `wiredMsg` for the round `"binance.tss-lib.ecdsa.keygen.KGRound3Message"`, finally, the Tss process stuck there.

**Comment by mungaij83** [comment]
This issue is affecting me as well. I can't move to round 2

**Comment by froyobin** [comment]
@notatestuser  Thank you for your reply, that make sense. I was thinking bout the scenario that the malicious nodes tamper the share and trying to send incorrect tss share to the victim with the `session ID` of this round.

**Comment by notatestuser** [comment]
@froyobin yeah. I think what you are looking for is covered in section 4.1 of the [2020 paper](https://eprint.iacr.org/2020/540.pdf). The paper from 2018 that this lib implements does not provide identifiable aborts in keygen.

**Comment by froyobin** [comment]
@notatestuser  Thank you. I will have a try with that one.
